Cargo.toml

```diff
@@ -70,22 +70,22 @@ version = "42.1.0"
 ahash = { version = "0.8", default-features = false, features = [
     "runtime-rng",
 ] }
-arrow = { version = "53.1.0", features = [
+arrow = { version = "53.2.0", features = [
```

Comment:
this update is required to have access to https://github.com/apache/arrow-rs/pull/6539